MSI-1631: Unskipping Tests

ivy00johns opened the pull request:
### Description
Unskipping the following Tests:
- MSI-1577 - AdminVirtualProductDisabledManageStockOnCustomStockTest
- MSI-1316 - AdminCreateOrderWithGroupedProductDefaultStockTest
- MSI-1313 - AdminOrderCreatedWithConfigurableProductWithDropDownAttributeViaTheAdminTest
- MSI-1312 - AdminOrderCreatedWithConfigurableProductWithSwatchAttributeViaTheAdminTest
- MSI-1325 - AdminOrderCreatedWithConfigurableProductWithTextAttributeViaTheAdminTest
- MSI-1008 - AdminOrderCreatedWithSimpleProductViaTheAdminTest

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

rogyar committed:
Added HTML renderer for description and short description

rogyar committed:
Api-functional tests added

rogyar committed:
Check that rendered values do not contain directives

chrom committed:
GraphQl-129: Retrieve Customer token

rogyar committed:
Quote masked id creation logic moved to create cart resolver

pfantini committed:
GraphQL-129: Change generate token schema and add graphql exception

Commit:
Merge remote-tracking branch 'origin/2.3-develop' into hashed-cart-create-refactor

mage2pratik committed:
Replace floatval() function by using direct type casting to (float)

pfantini committed:
GraphQL-129. Add generate customer token test

pfantini committed:
GrapgQL-129: Add composer dependency and fix codestyle issues

TBlindaruk committed:
[2.3] Changed intval($val) to (int) $val, since it is faster:
 - changed intval($val) to (int) $val, since it is faster

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Unskipping Test #1319 after a minor scrolling of the page correction.

yaroslav-zenin committed:
typofix: ImportCollection -> ItemCollection

magento-engcom-team committed:
ENGCOM-2968: [2.3] Changed intval($val) to (int) $val, since it is faster: #18035

 - Merge Pull Request magento/magento2#18035 from TBlindaruk/magento2:changed_intval_to_int_v2
 - Merged commits:
   1. 90c5387a15ef51c146cc0bc40585ee27af9b1445

magento-engcom-team committed:
ENGCOM-2970: Replace floatval() function by using direct type casting to (float) #18020

 - Merge Pull Request magento/magento2#18020 from mage2pratik/magento2:forwardport-patch-16848
 - Merged commits:
   1. fd0e8b383ce7dd586465d450f74d14dfcde4b52f

rogyar committed:
Added integration tests for gift message quote merge

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Adding the correct magentoCLI action to set the necessary config.
- Unskipping the Test.

mage2pratik committed:
Code improvement of lib files

pfantini committed:
GraphQL-129: Change resolve to return array, update schema and test

magento-engcom-team committed:
ENGCOM-2972: Code improvement of lib files #18049

 - Merge Pull Request magento/magento2#18049 from mage2pratik/magento2:code-improvement-7
 - Merged commits:
   1. 5ef0489ac2504bd089e3f176fa5995a1eb90ef92

yaroslav-zenin committed:
Merge branch '2.3-develop' of github.com:magento/magento2 into 2.3-develop

TomashKhamlai committed:
Remove PHPMD directive

TomashKhamlai committed:
Merge remote-tracking branch 'origin/2.3-develop' into recheck-129-retrive-customer-token

TomashKhamlai committed:
Simplify code to avoid complexity of getting just one string

TomashKhamlai committed:
Remove usage of Value Factory and optimize imports

magento-engcom-team committed:
ENGCOM-2979: Added integration test for gift message quote merge #18047

 - Merge Pull Request magento/magento2#18047 from rogyar/magento2:2.3-quote-merge-giftmessage-it
 - Merged commits:
   1. e2d96ae6a374eec4cba5a4418f2274067633c791

TomashKhamlai committed:
Add checks for input arguments

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Adding the correct magentoCLI action to set the necessary config.
- Unskipping the Test.
- Removing duplicate test.

pfantini committed:
GraphQL-80: rename canonical_url to relative_url and fix covered test

pfantini committed:
GraphQL-80: remove extra lines

dmytro-ch committed:
Sales: add missing unit tests for model classes

Commit:
GraphQl-44: added schema for format of textarea ui element

Signed-off-by: vitaliyboyko <v.boyko@atwix.com>

Commit:
Merge branches '2.3-develop' and 'rendered-attributes-product' of https://github.com/magento/graphql-ce into rendered-attributes-product

magento-engcom-team committed:
ENGCOM-2984: Sales: add missing unit tests for model classes #18072

 - Merge Pull Request magento/magento2#18072 from dmytro-ch/magento2:2.3-unit-test-sales-model
 - Merged commits:
   1. ec7e74cbcd1f30f94a0aeed5d468a26cc1994081

Commit:
[Revert]GraphQl-44: added schema for format of textarea ui element

Will be added in scope of further PR's

Signed-off-by: vitaliyboyko <v.boyko@atwix.com>

magento-engcom-team committed:
ENGCOM-2969: typofix: ImportCollection -> ItemCollection #18043

 - Merge Pull Request magento/magento2#18043 from yaroslav-zenin/magento2:2.3-develop
 - Merged commits:
   1. 608d825eba3c976b19cfe49d77e4ac9fbc4c63ae
   2. 2051521670984913686d21d702b21cb1cfa9a067

Commit:
ENGCOM-2984: Sales: add missing unit tests for model classes #18072

Commit:
ENGCOM-2979: Added integration test for gift message quote merge #18047

Commit:
ENGCOM-2972: Code improvement of lib files #18049

Commit:
ENGCOM-2970: Replace floatval() function by using direct type casting to (float) #18020

Commit:
ENGCOM-2968: [2.3] Changed intval($val) to (int) $val, since it is faster: #18035

Commit:
ENGCOM-2969: typofix: ImportCollection -> ItemCollection #18043

Commit:
GraphQL-140: Added HTML renderer for description and short description

magento-engcom-team committed:
ENGCOM-2991: GraphQL-80: rename canonical_url to relative_url and fix covered test #185

 - Merge Pull Request magento/graphql-ce#185 from pfantini/graphql-ce:80-urlResolver-rename-canonical_url-to-relative_url
 - Merged commits:
   1. c7635146f3d70e8b69230ac8d89493d88367cd4d
   2. 184f90b495ddf0475a34adab7a53250405ba562c

Commit:
Merge remote-tracking branch 'origin/2.3-develop' into recheck-129-retrive-customer-token

magento-engcom-team committed:
ENGCOM-2989: Recheck 129 retrive customer token #184

 - Merge Pull Request magento/graphql-ce#184 from magento/graphql-ce:recheck-129-retrive-customer-token
 - Merged commits:
   1. 527e4febe49ff6c2ab3b42ddabf798307f0e7222
   2. ea9b2a8a73da00df665da807b830615c129945fc
   3. ca0c135c56511a3b67319f54f1ce39894c764e55
   4. c0d1b7b7400dd53e78e15b42d68c6bf4df46b195
   5. e97301d483ed196eef7b7ac33f3a26bc1d737ed9
   6. 85cd23901cd2e4cbdd184a172358314f7094777e
   7. 6a250c182293b6e209b78500ae771ffbeced570f
   8. 21e719c2f7062652b8b1d564474bac1426242633
   9. ad388f6e1ed3ee97c27220066938c9efa9561837
   10. 277fe084c5af746dbdc254d9ffb5929c175a0848
   11. 5dce3bd8eeb8c93b5cb347cd9d2dcb9f2e688bae

Commit:
Merge remote-tracking branch 'origin/2.3-develop' into rendered-attributes-product

Commit:
GraphQL-169: Quote masked id creation logic moved to create cart resolver

magento-engcom-team committed:
ENGCOM-2994:  Quote masked id creation logic moved to create cart resolver #169

 - Merge Pull Request magento/graphql-ce#169 from magento/graphql-ce:hashed-cart-create-refactor
 - Merged commits:
   1. 8771f41e6191d3a7e26cca0413740524f1106b23
   2. 8b6aab66d7e34a2794a5e247545cdc37375d0d14
   3. afa216974bf1214ee860e7bb9eeaf0474aee1d4f

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Removing duplicate command in a test.

magento-engcom-team committed:
[EngCom] Public Pull Requests - 2.3-develop

 - merged latest code from mainline branch

magento-engcom-team committed:
ENGCOM-2990: Added HTML renderer for description and short description #140

 - Merge Pull Request magento/graphql-ce#140 from magento/graphql-ce:rendered-attributes-product
 - Merged commits:
   1. 0cdfdcb5676bd24a3ab2562ddd59501868c887c1
   2. 6638f7eb8a5754ca71e33220d468f365c98c6fa4
   3. 4fbb2aee99f5e1295f00e7a71ab36049ecf0abb2
   4. df0d2ac5cfe4cb3cfc5333b4bfd254333a0a87d5
   5. 4b14faa26ada4bd3e226937044254de3a9fd7b0a
   6. 506c1482629ca2f2a59adc7f99fff272c5a82ad8
   7. 854264d899b5cc1616d0889c8f67b96385716095
   8. 3f92af0cbc88784f77c58028d8a028f87a665fd2

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Moving <skip> tags to the SkippedTest file. I will use this to run tests in the B2B Jenkins build.

Commit:
magento-engcom/magento2ce#2180: Code style fixes

Commit:
ENGCOM-2994:  Quote masked id creation logic moved to create cart resolver #169

Commit:
ENGCOM-2991: GraphQL-80: rename canonical_url to relative_url and fix covered test #185

Commit:
ENGCOM-2990: Added HTML renderer for description and short description #140

Commit:
ENGCOM-2989: Recheck 129 retrive customer token #184

magento-engcom-team committed:
[EngCom] Public Pull Requests - 2.3-develop

 - merged latest code from mainline branch

magento-engcom-team committed:
[EngCom] Public Pull Requests - 2.3-develop

 - merged latest code from mainline branch

ishakhsuvarov committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.3-develop

Accepted Public Pull Requests:
 - magento/magento2#18072: Sales: add missing unit tests for model classes (by @dmytro-ch)
 - magento/magento2#18047: Added integration test for gift message quote merge (by @rogyar)
 - magento/magento2#18049: Code improvement of lib files (by @mage2pratik)
 - magento/magento2#18020: Replace floatval() function by using direct type casting to (float) (by @mage2pratik)
 - magento/magento2#18035: [2.3] Changed intval($val) to (int) $val, since it is faster: (by @TBlindaruk)
 - magento/magento2#18043: typofix: ImportCollection -> ItemCollection (by @yaroslav-zenin)

magento-engcom-team committed:
[EngCom] Public Pull Requests - 2.3-develop

 - merged latest code from mainline branch

Commit:
Merge pull request #3183 from magento-engcom/graphql-develop-prs

[EngCom] Public Pull Requests - GraphQL

IvanPletnyov committed:
Merge remote-tracking branch 'm2ce/2.3-develop' into merge-ce-2.3-develop

# Conflicts:
#	composer.lock
#	dev/tests/integration/testsuite/Magento/Quote/Model/QuoteTest.php
#	dev/tests/static/testsuite/Magento/Test/Integrity/Magento/Backend/_files/controller_acl_test_whitelist_ce.txt

IvanPletnyov committed:
Update composer lock

maghamed committed:
Merge pull request #1656 from magento-engcom/merge-ce-2.3-develop

Merge CE 2.3 develop into MSI

IvanPletnyov committed:
MSI-1655: 404 error after source selection during creating shipment

maghamed committed:
Merge pull request #1657 from magento-engcom/MSI-1655

MSI-1655: 404 error after source selection during creating shipment

maghamed committed:
MSI-1659: Remove FK from inventory_low_stock_notification_configuration table

okolesnyk committed:
magento-engcom/msi#1631: stabilize MSI test suites

maghamed committed:
Merge pull request #1661 from magento-engcom/MSI-1659

MSI-1659: Remove FK from inventory_low_stock_notification_configurati…

okolesnyk committed:
magento-engcom/msi#1631: stabilize MSI test suites

okolesnyk committed:
magento-engcom/msi#1631: stabilize MSI test suites

okolesnyk committed:
Merge branch '2.3-develop' into MFTF-issues

okolesnyk committed:
Merge pull request #1642 from magento-engcom/MFTF-issues

MFTF tests issues

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Moving <skip> tags to the SkippedTest file. I will use this to run tests in the B2B Jenkins build.
Adding space to seperate Tests.

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Unskipping 3 Tests. Locally they run correctly individually and together.

ivy00johns committed:
Merge branch '2.3-develop' into 1631-Unskipping-Tests

okolesnyk commented on app/code/Magento/Inventory/Test/Mftf/Test/SkippedTest.xml:
```diff
@@ -8,6 +8,29 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="../../../../../../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Test/etc/testSchema.xsd">
+    <test name="AdminSourceSelectionAlgorithmSourcePriorityWithSimpleMsiProductTest">
```
Please unskip this 3 tests since as I can see the bugs was fixed

ivy00johns replied:
Done

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Unskipping 3 Tests.

okolesnyk commented:
https://m2build-ur.devops.magento.com/job/All-User-Requested-Tests/5146/

ivy00johns committed:
Merge branch '2.3-develop' into 1631-Unskipping-Tests

magento-cicd2 commented:
[![CLA assistant check](https://cla.dev.magento.com/pull/badge/not_signed)](https://cla.dev.magento.com/magento-engcom/msi?pullRequest=1663) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-engcom/msi?pullRequest=1663) before we can accept your contribution.<br/>**3** out of **10** committers have signed the CLA.<br/><br/>:white_check_mark: naydav<br/>:white_check_mark: ivy00johns<br/>:white_check_mark: TomashKhamlai<br/>:x: rogyar<br/>:x: chrom<br/>:x: pfantini<br/>:x: mage2pratik<br/>:x: TBlindaruk<br/>:x: yaroslav-zenin<br/>:x: magento-engcom-team

ivy00johns committed:
Merge branch '2.3-develop' into 1631-Unskipping-Tests

ivy00johns committed:
Merge branch '2.3-develop' into 1631-Unskipping-Tests

ivy00johns committed:
magento-engcom/msi#1631-Unskipping-Tests

- Skipping Test # 1135. Assigning it to the "sort_order_test" suite did not resolve the issue. More investigation needed.

ivy00johns committed:
Merge remote-tracking branch 'origin/1631-Unskipping-Tests' into 1631-Unskipping-Tests

ivy00johns committed:
Merge branch '2.3-develop' into 1631-Unskipping-Tests

ivy00johns commented:
Build: https://m2build-ur.devops.magento.com/job/All-User-Requested-Tests/5407/

ivy00johns committed:
Merge branch '2.3-develop' into 1631-Unskipping-Tests

okolesnyk commented:
https://m2build-ur.devops.magento.com/job/All-User-Requested-Tests/5784/

okolesnyk commented:
@ivy00johns 
Your branch contains invalid commits. Please create new branch based on `2.3-develop` and apply only your changes on top of it.
And create new PR.
